runtime/src/bank.rs

```diff
@@ -5375,14 +5375,18 @@ impl Bank {
         true
     }
 
+    fn prefix_from_pubkey(pubkey: &Pubkey) -> u64 {
```

Comment:
Wondering if a more natural place for this is the pubkey.rs file? Not sure if the pubkey prefix is a concept we use outside of bank though, so I'll leave it to your judgement.